modules/deb.sh

```diff
@@ -47,7 +47,8 @@ function deb_from_repo_dist_get()  {
     fi
 
     local -r filename=$(basename $deb_package_path)
-    run_and_log_cmd "wget -Nq ${repo_url}/${deb_package_path} -P ${download_dir}"
+    local -r deb_package_url=$(echo ${repo_url}/${deb_package_path} | sed 's/+/%2B/g')
+    run_and_log_cmd "wget -Nq ${deb_package_url} -P ${download_dir}"
```

Comment:
Fantastic! Nice catch.

This bug has been hidden for a while because our pre-converted image did not install any-more the client (hence the `versions_override_config` is ignored, this by the way could be cleaned up) and the rest of the conversions were using default (latest).

This commit requires a changelog, though (it fixes installing "master" packages) and a cherry-pick to release branches.

Reply:
I think you are right. But thinking about it, there is something weird going on. Why did this succeed for Raspberry Pi (production), but fail for the others (only tests)?

Reply:
Ah, wait, it's because the package is not installed, of course. Then never mind, I'm fixing.